### ISSUE TITLE
[Package Request]: AMD Radeon Graphics Drivers

### ISSUE THREAD

**Issue by khewweifeng** [microsoft/winget-pkgs#90983]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: 
Publisher Url: https://www.techpowerup.com/download/amd-radeon-graphics-drivers/

Package Name: AMD Radeon Graphics Drivers

Description: Download the latest AMD Radeon drivers for best game performance, compatibility and to fix driver issues.

Radeon Adrenalin comes with a completely redesigned interface. This download supports the following AMD Radeon graphics cards:

AMD RX 6500 / 6600 XT / 6700 XT / 6750 XT / RX 6800 / 6800 XT / 6900 XT / 6950 XT
AMD RX 5700 / 5600 / 5500 Navi
Radeon VII / Radeon RX Vega
AMD Radeon RX 580 / 570 / 560, RX 480 / 470 / 460
All AMD Radeon R9 / R5 / R3
and many more, incuding older models all the way back to the Radeon HD Series

Package Version: 22.11.2 WHQL
Installer URL:  https://us2-dl.techpowerup.com/files/gfbQr6RpjgnRmafS2Tmd-g/1670690305/whql-amd-software-adrenalin-edition-22.11.2-win10-win11-dec8.exe

Current Package Identifier (for new versions):

**Comment by Trenly** [comment]
I believe this is blocked for a few reasons -
1) I can't seem to find a silent switch
2) According to the files inside the exe, it is dependent upon the hardware installed

![image](https://user-images.githubusercontent.com/12611259/206830745-b6a0c80c-0921-46e0-a25e-533ab439a60a.png)

@msftbot interactive-only
@stephengillie - Please also tag hardware

**Comment by R-Adrian** [comment]
That techpowerup article is wrong, AMD has (recently) discontinued driver support for a lot of older hardware, in preparation for the launch of the current line of new GPUs.

e.g. for my 2-year old laptop that has a Radeon 530 GPU the "latest" driver is .... 22.6.1 and i get the message "This product has been moved to a legacy support model and no additional driver releases are planned."

![image](https://user-images.githubusercontent.com/16319504/206896964-ad715781-56b6-4542-a1a7-89ddd44021b9.png)

**Comment by Chudesnov** [comment]
While the issue with hardware-dependent installer still exists, the "Interactive-Only-Installer" label is misleading, AMD has a documented `-INSTALL` flag for silent installation: https://www.amd.com/system/files/documents/radeon-software-command-line-installation-user-guide.pdf

Example for the latest currently available version (24.3.1) for RDNA1+ based products:

```bat
whql-amd-software-adrenalin-edition-24.3.1-win10-win11-mar20-rdna.exe -INSTALL
```

**Comment by RokeJulianLockhart** [comment]
> https://github.com/microsoft/winget-pkgs/issues/90983#issuecomment-2550047946

@denelon, does that also apply to https://github.com/microsoft/winget-pkgs/issues/90511#issuecomment-2909858274?
